bin/phpunit

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env php
+<?php
+
+if (!file_exists(dirname(__DIR__).'/vendor/symfony/phpunit-bridge/bin/simple-phpunit')) {
+    echo "Unable to find the `simple-phpunit` script in `vendor/symfony/phpunit-bridge/bin/`.\n";
+    exit(1);
+}
+
+if (false === getenv('SYMFONY_PHPUNIT_DIR')) {
+    putenv('SYMFONY_PHPUNIT_DIR='.__DIR__.'/.phpunit');
+}
+
```

Comment:
:information_source: Per paskaitą buvo rodomas pavyzdys su naujausia PHPUnit versija:
https://github.com/aurelijusb/kickstart/blob/73db76773d9bc3971b76984b323ffaf614b83175/bin/phpunit#L14

Naudojant senesnę:
![image](https://user-images.githubusercontent.com/1453957/58270464-90d3af00-7d92-11e9-8575-8e3bbeba52cc.png)

Gali būti [sunkiau](https://phpunit.readthedocs.io/en/6.5/writing-tests-for-phpunit.html) rasti dokumentaciją